package.json

```diff
@@ -31,29 +31,36 @@
     "html"
   ],
   "devDependencies": {
-    "@babel/core": "^7.10.4",
-    "@babel/preset-env": "^7.10.4",
-    "@markedjs/html-differ": "^3.0.2",
+    "@babel/core": "^7.11.6",
+    "@babel/preset-env": "^7.11.5",
+    "@markedjs/html-differ": "^3.0.3",
+    "@semantic-release/changelog": "^5.0.1",
+    "@semantic-release/commit-analyzer": "^8.0.1",
+    "@semantic-release/git": "^9.0.0",
+    "@semantic-release/github": "^7.1.1",
+    "@semantic-release/npm": "^7.0.6",
+    "@semantic-release/release-notes-generator": "^9.0.1",
```

Comment:
That's a lot of extra dependencies 😧 

Do we need all of these? For example, `@semantic-release/changelog` seems unnecessary since we don't keep a changelog. I think `@semantic-release/release-notes-generator` is sufficient.

Reply:
I didn't even realize we didn't have a changelog.md

Reply:
Yeah. These dependencies are not needed! You can just install them in the CI if not included in the action

https://github.com/atom-ide-community/atom-ide-datatip/blob/fe61eec0db91901ef85f2f3eb42c27ae3820c5b1/.github/workflows/ci.yml#L75